model/error/event.go

```diff
@@ -192,12 +192,18 @@ func (e *Event) Transform(tctx *transform.Context) []beat.Event {
 		"processor": processorEntry,
 	}
 	delete(e.Context, "user")
-	utility.Add(fields, "context", e.Context)
 	utility.Add(fields, "user", e.User.Fields())
 	utility.Add(fields, "client", e.User.ClientFields())
 	utility.Add(fields, "user_agent", e.User.UserAgentFields())
 	tctx.Metadata.Merge(fields)
 
+	utility.Add(fields, "http", m.HttpFields(e.Context))
+	utility.Add(fields, "url", m.UrlFields(e.Context))
+	delete(e.Context, "request")
+	delete(e.Context, "response")
+
+	utility.Add(fields, "context", e.Context)
```

Comment:
If I understand this PR correct, then it aims for getting rid of `context`. What's left here?